src/lib/stores/AuthStore.js

```diff
@@ -71,9 +71,26 @@ class AuthStore {
     return false;
   }
 
+  // TODO: Temporary workaround until name fields get added from GQL
+  // See https://github.com/reactioncommerce/reaction/issues/4646
+  splitNames(account) {
+    const { name } = account;
+    const [firstName, lastName] = name.split(" ");
```

Comment:
@mike probably from this? 
Maybe something like `const [firstName, lastName] = name && name.split(" ");`

Reply:
Okay. Will push a fix